### ISSUE TITLE
Pa11y fails with error URL could not be loaded

### ISSUE THREAD

**Issue by rajesh825** [nature/pa11y#64]
#Pa11y Ignore SSL Certificate warning

Hello, I am trying to integrate Pa11y as part of Build process. However on trying to perform site validation seeing following error.  

/pa11y WCAG2AA https://testsiteurl

_-::>  Loading page...
_-::>  URL could not be loaded

It appears my Local test environment does not have valid SSL Certificate. 
for eg: Curl https://testsiteurl throws following message 
curl: (60) SSL certificate problem: Invalid certificate chain
 
Could you please advise, if there is any way to ignore ssl certificate warning, when validating urls with Pa11y.
for eg: Curl -k  testsiteurl , using -k we can ignore ssl warning with Curl

Thanks,
Raja

**Comment by rowanmanning** [comment]
Hi Raja, thanks for reporting. I'll look into this as soon as possible

**Comment by whymarrh** [comment]
To ignore SSL errors (and/or to allow self-signed certs) you'd need to pass a flag to PhantomJS itself, something that isn't yet exposed through Pa11y. The flag you're looking for is `--ignore-ssl-errors=true`<sup>[1](http://phantomjs.org/api/command-line.html)</sup>. You might be able to get away with using @mfairchild365's branch from #55, but do note that its behind master.

**Comment by rajesh825** [comment]
That's great, thanks for the information.
That was useful.

Regards,
Rajesh
